lib/fs-driver.js

```diff
@@ -30,7 +30,9 @@ class FsDriver {
     if (!isString(str)) {
       str = str.toString()
     }
-    return str.replace(/(['/\\])/g, '\\$1')
+    return str.replace(/'/g, '\'"\'"\'')
```

Comment:
So roughly what this is doing is using juxtaposition in the command to "emulate" quote escaping. The first `'` ends the single quote, the next sequence is `"'"` which wraps double quotes around a single quote so it is output as is, then then final `'` opens the single-quote again.